ci/dev-pipeline.yml

```diff
@@ -31,7 +31,9 @@ env_setups:
 
   contentful_creds: &contentful_creds
     SPACE_ID: ((contentful_space_id))
-    CDA_KEY: ((contentful_delivery_api_key))
+    CDA_DEV_KEY: ((contentful_delivery_api_key_dev))
```

Comment:
Could we have kept CDA_KEY but set it to the appropriate value under sml_dev, sml_preprod and sml_prod below?  By default the pipeline sets the key appropriate to the environment and then you don't need the json environment variable?

Reply:
This should work, my only concern is that we might lose the ability to locally test the differences of content environments. I'll take another look though